docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,162 @@
+
+{{% notice tip %}}
+If you just want to create deployment topologies that include all available clusters you are not required to choose labels at installation time. 
+All virtual nodes expose the label `liqo.io/type = virtual-node` by default.
+{{% /notice %}}
+
+### NamespaceOffloading in quick offloading
+
+Also the *quick Offloading* relies on the namespaceOflloading resource. 
+In fact, when the label `liqo.io/enabled = true` is added to a namespace, this event triggers the creation of a default *namespaceOffloading* resource inside the namespace.
+
+The generated resource is exactly equal to the template [seen above](#custom-offloading), setting all fields to the default values:
+
+* **namespaceMappingStrategy**: *DefaultName*.
+* **podOffloadingStrategy**: *LocalAndRemote*.
+* **clusterSelector**: *all remote clusters selection*.
+
+{{% notice info %}}
+Using the *quick offloading*, it is not possible to customize the generated *namespaceOffloading* resource. 
+If a *namespaceOffloading* is already present in the namespace, you should remove the label first and then create the new resource.
+Removing the label is equivalent to removing the resource directly.
+{{% /notice %}}
+
+{{% notice tip %}}
+If you create a custom *namespaceOffloading* inside the namespace, the `liqo.io/enabled = true` label is not necessary.
+As previously said, this label is just a way to configure a *quick offloading*. 
+{{% /notice %}}
+
+### Check the namespaceOffloading status
+
+The Liqo controllers update the *namespaceOffloading* every time there is a change in the deployment topology. 
+The resource status provides different information:
+
+```bash
+kubectl get namespaceoffloading offloading -n test-namespace -o wide
+```
+
+The global offloading status (**OffloadingPhase**) can assume different values:
+
+| Value                 | Description |
+| --------------        | ----------- |
+| **Ready**             |  Remote Namespaces have been correctly created inside previously selected clusters. |
+| **NoClusterSelected** |  No cluster matches user constraints or constraints are not specified with the right syntax (in this second case, an annotation is also set on the namespaceOffloading, specifying what is wrong with the syntax)        |
+| **SomeFailed**        |  There was an error during some remote namespaces creation. |
+| **AllFailed**         |  There was an error during all remote namespaces creation. |
+| **Terminating**       |  Remote namespaces are undergoing graceful termination. |
+
+Obviously, the **remoteNamespaceName** should match the *namespaceMappingStrategy* chosen in the resource spec.
+
+### Offloading termination
+
+To terminate the offloading, you can remove the *namespaceOffloading* resource inside the namespace or, in case of *quick offloading*, the previously inserted label `liqo.io/enabled = true`.
```

Comment:
```suggestion
To terminate the offloading, you can remove the *namespaceOffloading* resource inside the namespace or, in case of the *quick offloading* approach, the previously inserted label `liqo.io/enabled = true`.
```